docs/ROADMAP.md

```diff
@@ -7,38 +7,54 @@ category: Roadmap
 
 This document defines the roadmap for TiKV development.
 
+## Engine
+- [ ] Titan - Use the separated key-value engine in production
+- [ ] Pluggable Engine Interface - Clean up the engine wrapper code and provide more extendibility
+
 ## Raft
-- [x] Region Merge - Merge small Regions together to reduce overhead
```

Comment:
It's odd to remove finished items 🤣 letting people thinking that every thing is unfinished.

Reply:
this is the roadmap for 2018, and now it is for 2019

maybe we can move old to roadmap-2018.md